### ISSUE TITLE
Packages installed with pip cannot be used in non-root environment

### ISSUE THREAD

**Issue by maxseidler** [mandeep/sublime-text-conda#3]
I have Python 3 configured as root and Python 2 configured in an extra environment. In Python 3 / root I can import/use packages both installed with conda and with pip.

In the Python 2 environment modules installed with pip are not recognised and cannot be used.

**Extra suggestion:**
`Conda: List Packages` does not currently show packages installed with pip, neither in root or other environments. Executing `conda list` in the Terminal *is* showing pip-installed packages as well though. Maybe consider to list pip-installed packages when executing `Conda: List Packages` in sublime-text-conda as well?

**Comment by mandeep** [comment]
Thanks for reporting this. It looks like the ``--json`` flag on conda does not report those pip installed packages. I'll open an issue on the conda issue tracker for this.

**Comment by mandeep** [comment]
@maxseidler Using pip installed packages in Python 2 environments seem to work fine for me. Can you give me more details about your environment?

**Comment by maxseidler** [comment]
Actually, between the first occurrence of this issue and now, I restarted my machine... and now I *can* use pip-installed packages in my Python 2 environment! I tried a couple of pip-installed packages before I wrote this issue and they all failed. Anyway, it works and the previous hick-up was probably no fault of sublime-text-conda. But at least the discovery of the  '--json flag' bug came out of it. ;)

Thanks for the support!
